### PR TITLE
Update CONTRIBUTING.md for Hacktoberfest

### PR DESCRIPTION
**Pull request by Luna-meili**
# Pull Request

## What does this PR do?
Edit to include Hacktoberfest 2022 to our contributing file

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue?
- [ ] Have you read the contributing guidelines?
- [ ] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,12 +10,22 @@ If Meilisearch does not offer optimized support for your language, please consid
...
 - [Development Workflow](#development-workflow)
 - [Git Guidelines](#git-guidelines)
 - [Release Process (for internal team only)](#release-process-for-internal-team-only)
 
+## Hacktoberfest 2022
+
+It's [Hacktoberfest month](https://hacktoberfest.com)! 🥳
+
+Thanks so much for participating with Meilisearch this year!
+1. We will follow the quality standards set by the organisers of Hacktoberfest (see detail on their [website](https://hacktoberfest.digitalocean.com/resources/qualitystandards)). Our reviewers will not consider any PR that doesn’t match that standard.
```
Comment (brunoocasali) [inline CONTRIBUTING.md:25]:
```suggestion
1. We will follow the quality standards set by the organizers of Hacktoberfest (see detail on their [website](https://hacktoberfest.digitalocean.com/resources/qualitystandards)). Our reviewers will not consider any PR that doesn’t match that standard.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -10,12 +10,22 @@ If Meilisearch does not offer optimized support for your language, please consid
...
+2. PRs reviews will take place from Monday to Thursday, during usual working hours, CEST time. If you submit outside of these hours, there’s no need to panic; we will get around to your contribution.
+3. There will be no PR assignment as we don’t want people to ask to be assigned specific PRs and never return, discouraging the volunteer contributors from opening a PR to fix this issue. We take the liberty to chose the PR that best fixes the issue, so we encourage you to get to it as soon as possible and do your best!
```
Comment (brunoocasali) [inline CONTRIBUTING.md:27]: We want to say that we will not assign people to issues, right?
```suggestion
3. There will be no issue assignment as we don’t want people to ask to be assigned specific issues and never return, discouraging the volunteer contributors from opening a PR to fix this issue. We take the liberty to choose the PR that best fixes the issue, so we encourage you to get to it as soon as possible and do your best!
```

**Review by brunoocasali — CHANGES_REQUESTED** [review]
I found some words that are not correct according to Grammarly :p 

But also I would like to see about the last item, I think we want to say that phrase about the issue not about PRs :)

File: CONTRIBUTING.md (hunk 1 above)
Comment (Luna-meili) [reply CONTRIBUTING.md:25]: You don't like my British spelling :p
Happy with the changes on my side

File: CONTRIBUTING.md (hunk 2 above)
Comment (Luna-meili) [reply CONTRIBUTING.md:27]: Ah yes yes yes exactly, apologies if I made it more difficult that it needed to be

**Review by brunoocasali — APPROVED** [review]
Awesome job @Luna-meili 🎉

**Review by curquiza — APPROVED** [review]
Thanks for this @Luna-meili

bors merge!

**Comment by curquiza** [comment]
I merge by hand, because this PR is not related to code base, and we need it

**Pull request merged**